templates/enmasse-operator/020-ClusterRole-enmasse-operator.yaml

```diff
@@ -21,3 +21,9 @@ rules:
     resources: [ "apiservices" ]
     resourceNames: [ "v1alpha1.enmasse.io", "v1beta1.enmasse.io", "v1alpha1.user.enmasse.io", "v1beta1.user.enmasse.io" ]
     verbs: [ "create", "update", "patch", "get", "list", "watch", "delete" ]
+  - apiGroups: [ "monitoring.coreos.com" ]
+    resources: [ "prometheusrules", "servicemonitors" ]
```

Comment:
The same permissions must be added to olm-manifest/src/main/resources/latest/enmasse.clusterserviceversion.yaml so that the OLM operator works.